megamek/src/megamek/server/GameManager.java

```diff
@@ -2762,6 +2762,13 @@ private void determineTurnOrderIUI(GamePhase phase) {
         } else {
             entities = game.getEntitiesVector();
         }
+
+        String noInitiative = entities.stream().filter(e -> e.getInitiative().size() == 0).map(Object::toString).collect(Collectors.joining(";"));
+
+        if (!noInitiative.isEmpty()) {
+            LogManager.getLogger().error("No Initiative rolled for: " + noInitiative);
+        }
```

Comment:
Should we escape or exit out here?

Reply:
it mostly seems to work if there there is no initiative is assigned.   just seems to error when it  interacts with the other initiatives just right.   just putting this in to hopefully help if somewhere else is not assigning it.

also just escaping or erroring here would probably break the turn, since nothing would be assigned to the turn object.

Reply:
i tried filtering out the ones with no initiative, but it was then causing errors later when checking for the turn in the entities.